search-website-functions-v4/api/Models.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Azure.Search.Documents.Models;
+using System.Text.Json.Serialization;
+
+
+namespace WebSearch.Models

```

Comment:
Consider using file-scoped namespaces too, that really helps with readability and helps to alleviate potential horizontal scrollbars. https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-10#file-scoped-namespace-declaration